tools/compile_tf_graph.py

```diff
@@ -612,16 +766,32 @@ def cond(i, net_vars, acc_tas, seq_len_info=None)
       seq_len_info = None
     else:
       i, net_vars, acc_tas, seq_len_info = loop_vars
-      seq_len_info = rec_layer.create_state_vars_recursive(["end_flag", "dyn_seq_len"], seq_len_info)
-    i = rec_layer.create_state_var("i", i)
```

Comment:
Please leave that in. Also, why did you move `create_state_var("i", i)` down? Leave it just right here.

Reply:
I want the end_flag to be always false and dyn_seq_len always 0 as they will be now. Save mem from whatever is useless to be a stat_var

Reply:
This takes 8 bytes of memory, so really you can ignore that. Also, there are models which can potentially depend on that, and that would make it more complicated then.

Reply:
Also this is needed.